src/Sarif.Converters/PREFastConverter.cs

```diff
@@ -0,0 +1,193 @@
+﻿using System.IO;
+using Microsoft.CodeAnalysis.Sarif.Writers;
+using System;
+using System.Collections.Generic;
+using System.Xml;
+using System.Xml.Serialization;
+using Microsoft.CodeAnalysis.Sarif.Converters.PREFastObjectModel;
+using System.Linq;
+
+namespace Microsoft.CodeAnalysis.Sarif.Converters
+{
+    public class PREfastConverter : ToolFileConverterBase
+    {
+        private Dictionary<string, string> categories = new Dictionary<string, string>
```

Comment:
Seeing how this is used below, should it be named something like "`knownCategories`"?

Reply:
fixed

---
In reply to: [146950815](https://github.com/Microsoft/sarif-sdk/pull/657#discussion_r146950815) [](ancestors = 146950815)